app/components/supplier_table_component.html.haml

```diff
@@ -8,6 +8,12 @@
         %th{ scope: "col" }
           .cads-table__content
             Supplier
+        %th{ scope: "col" }
+          .cads-table__content
+            .cads-hide-md-up
+              Overall rating
+            .cads-show-md-up
+              Overall rating
```

Comment:
If the change is to make the column heading `Overall rating` at all times, we don't need the different show and hide divs, so we can simplify this to just having the text once directly inside the `.cads-table__content` div